tests/func/api/test_data.py

```diff
@@ -75,7 +75,7 @@ def test_missing(tmp_dir, dvc, remote):
 
     remove("foo")
 
-    with pytest.raises(FileMissingError):
+    with pytest.raises(PathMissingError):
```

Comment:
This was a hidden inconsistency. If using local repo - you'd get `FileMissingError`, but you'd get `PathMissingError` with everything else. Going with the assumption that latter is more common so making this consistent